### ISSUE TITLE
kokoro clang-format check erroring out but passing

### ISSUE THREAD

**Issue by dneto0** [KhronosGroup/SPIRV-Tools#1676]
See https://source.cloud.google.com/results/invocations/9151a1a8-3873-457a-b747-63ad3c860d21/log

**Comment by dneto0** [comment]
```
++ date
+ echo Fri Jul 6 10:38:08 PDT 2018: Check formatting...
Fri Jul 6 10:38:08 PDT 2018: Check formatting...
+ ./utils/check_code_format.sh
Traceback (most recent call last):
  File "./utils/clang-format-diff.py", line 121, in <module>
    main()
  File "./utils/clang-format-diff.py", line 104, in main
    stderr=None, stdin=subprocess.PIPE)
  File "/usr/lib/python2.7/subprocess.py", line 710, in __init__
    errread, errwrite)
  File "/usr/lib/python2.7/subprocess.py", line 1327, in _execute_child
    raise child_exception
OSError: [Errno 2] No such file or directory
All source code in PR properly formatted.
++ date
```

**Comment by ehsannas** [comment]
fixed via #1710
